validator/valnode/redis/consumer_test.go

```diff
@@ -0,0 +1,29 @@
+package redis
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/offchainlabs/nitro/util/redisutil"
+	"github.com/offchainlabs/nitro/util/testhelpers"
+)
+
+func TestTimeout(t *testing.T) {
+	handler := testhelpers.InitTestLog(t, log.LevelInfo)
+	ctx, cancel := context.WithCancel(context.Background())
+	redisURL := redisutil.CreateTestRedis(ctx, t)
+	TestValidationServerConfig.RedisURL = redisURL
+	TestValidationServerConfig.ModuleRoots = []string{"0x123"}
+	vs, err := NewValidationServer(&TestValidationServerConfig, nil)
+	if err != nil {
+		t.Fatalf("NewValidationSever() unexpected error: %v", err)
+	}
+	vs.Start(ctx)
+	cancel()
+	time.Sleep(time.Second)
+	if !handler.WasLogged("Context done while waiting redis streams to be ready") {
```

Comment:
Context done messages are not very important, as the node is already shutting down..
I want a test making sure that the "Waiting for redis streams timed out" is printed, because we'll need that one.

Reply:
Done.

Reply:
you changed the error message but not the actual test?
the test shouldn't call contexxt.Cancel() before the wasLogged. Instead, it should program timeout to be very low and check that the timeout message was logged.
The test should fail if not changing the timeout.

Reply:
Oops, not sure how did I mess up the commit but I clearly remember changing the check text itself first without changing error message, committing, then coming back after an hour, changing only error message and committing. Must have done on other branch or something.

Thanks for catching!